client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/utils.ts

```diff
@@ -46,14 +46,13 @@ export function createCustomHomeTemplateContent(
 		);
 	}
 
-	content.push( `
-<!-- wp:group {"tagName":"main"} -->
-	<main class="wp-block-group">
-	<!-- wp:paragraph -->
-	<p></p>
-	<!-- /wp:paragraph -->
-	</main>
-<!-- /wp:group -->` );
+	if ( hasHeader || hasFooter ) {
```

Comment:
We have to check whether there is a selected section pattern. Otherwise, the sections won't be inserted into home template if none of header and footer is selected

Reply:
Updated. Thanks!

Reply:
I prefer to keep the paragraph, what do you think?

| Without paragraph | With paragraph |
| - | - |
| ![image](https://user-images.githubusercontent.com/13596067/205227083-d1d09490-5d01-48c7-890a-fb9fe0764e71.png) | ![image](https://user-images.githubusercontent.com/13596067/205227211-550297a8-767c-4e26-a5b6-b3afd57536e4.png) |

Reply:
I like it but we were told to keep it blank because it is a blank canvas.

Reply:
hmmm...okay